<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardAdmin</class>
 <widget class="QDialog" name="DashboardAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Dashboard - Job Matching System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #E6F0FA; /* Light blue gradient background */
        border: 1px solid #A3BFFA;
    }
    QLabel {
        font: 14px &quot;Arial&quot;;
        color: #2E4053; /* Dark slate color for text */
    }
    QTableWidget {
        background-color: #FFFFFF;
        border: 1px solid #A3BFFA;
        border-radius: 5px;
        font: 14px &quot;Arial&quot;;
    }
    QLineEdit {
        background-color: #FFFFFF;
        border: 1px solid #A3BFFA;
        border-radius: 5px;
        padding: 5px;
        font: 14px &quot;Arial&quot;;
    }
    QPushButton {
        background-color: #4CAF50; /* Vibrant green for buttons */
        color: white;
        border-radius: 5px;
        padding: 8px;
        font: 14px &quot;Arial&quot;;
    }
    QPushButton:hover {
        background-color: #45A049; /* Darker green on hover */
    }
    QPushButton#logoutButton {
        background-color: #FF6B6B; /* Red for logout */
    }
    QPushButton#logoutButton:hover {
        background-color: #FF5252; /* Darker red on hover */
    }
    QComboBox {
        background-color: #FFFFFF;
        border: 1px solid #A3BFFA;
        border-radius: 5px;
        padding: 5px;
        font: 14px &quot;Arial&quot;;
    }
   </string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>760</width>
     <height>560</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">
        font: bold 24px &quot;Arial&quot;;
        color: #2D6A4F; /* Rich green for title */
        background-color: #D1E8E2; /* Soft green background */
        border-radius: 5px;
        padding: 10px;
       </string>
      </property>
      <property name="text">
       <string>Admin Dashboard</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_1">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="statsLabel">
      <property name="styleSheet">
       <string notr="true">
        font: bold 18px &quot;Arial&quot;;
        color: #2E4053;
        background-color: #D1E8E2;
        padding: 5px;
       </string>
      </property>
      <property name="text">
       <string>System Statistics</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statsDisplayLabel">
      <property name="styleSheet">
       <string notr="true">
        font: 14px &quot;Arial&quot;;
        color: #2E4053;
        background-color: #FFFFFF;
        border-radius: 5px;
        padding: 5px;
       </string>
      </property>
      <property name="text">
       <string>Users: 0 | Jobs: 0 | Applications: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchSortLayout">
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="placeholderText">
         <string>Search by ID or Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortComboBox">
        <item>
         <property name="text">
          <string>Sort by ID (Asc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sort by ID (Desc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sort by Name (Asc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sort by Name (Desc)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="userTabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="employersTab">
       <attribute name="title">
        <string>Employers</string>
       </attribute>
       <layout class="QVBoxLayout" name="employersLayout">
        <item>
         <widget class="QTableWidget" name="employerTable">
          <property name="columnCount">
           <number>6</number>
          </property>
          <column>
           <property name="text">
            <string>User ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Company Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>License Number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>License Verified</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Blocked</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Action</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="employerActionLayout">
          <item>
           <widget class="QLineEdit" name="verifyUserIdLineEdit">
            <property name="placeholderText">
             <string>Enter Employer ID to Verify</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="licenseNumberLineEdit">
            <property name="placeholderText">
             <string>Enter License Number (e.g., ADMIN123)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="verifyButton">
            <property name="text">
             <string>Verify License</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="blockUserIdLineEdit">
            <property name="placeholderText">
             <string>Enter User ID to Block</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="blockButton">
            <property name="text">
             <string>Block User</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="jobSeekersTab">
       <attribute name="title">
        <string>Job Seekers</string>
       </attribute>
       <layout class="QVBoxLayout" name="jobSeekersLayout">
        <item>
         <widget class="QTableWidget" name="jobSeekerTable">
          <property name="columnCount">
           <number>6</number>
          </property>
          <column>
           <property name="text">
            <string>User ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Full Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Location</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Phone</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Blocked</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Action</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="jobSeekerActionLayout">
          <item>
           <widget class="QLineEdit" name="blockJobSeekerIdLineEdit">
            <property name="placeholderText">
             <string>Enter User ID to Block</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="blockJobSeekerButton">
            <property name="text">
             <string>Block User</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logoutButton</sender>
   <signal>clicked()</signal>
   <receiver>DashboardAdmin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
